connectivity/FEATURE_BLE/source/cordio/source/PalSecurityManagerImpl.cpp

```diff
@@ -592,11 +592,12 @@ bool PalSecurityManager::sm_handler(const wsfMsgHdr_t *msg)
     PalSecurityManager &self = get_security_manager();
     PalSecurityManagerEventHandler *handler = self.get_event_handler();
 
-    if ((msg == nullptr) || (handler == nullptr)) {
-        tr_warning("PAL Event handler: invalid message (%p) or handler (%p)", msg, handler);
+    if (handler == nullptr) {
```

Comment:
message cannot be null as it's already checked in stack_handler and the handler being null is not an error but normal operation and does not need logging as it causes spam